src/content/docs/alerts-applied-intelligence/notifications/message-templates.mdx

```diff
@@ -14,43 +14,48 @@ The features described here are early access. You won't be able to use these fea
 For more information on related features, see our docs on [Alerts notification channels](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts/), [Incident Intelligence destinations](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/incident-intelligence-destination-examples/), and [Proactive Detection notifications](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence/#set-up).
 </Callout>
 
-Notification message templates enable you to customize notification-event before they're sent by mapping custom values to your destinations. Doing so gives you full control over what data will be sent and where, as well as being able to fully engage with the services you use.
+Notification message templates enable you to customize your notification events data before it's sent to your third-party destination. The templates map your custom values to the values used by your third-party destination. 
```

Comment:
Is it `notification events data` or `notification event data`? Honestly not sure haha

Reply:
I think it's probably "notification event data" now that I think about it....